docker-compose.yml

```diff
@@ -124,8 +124,6 @@ services:
       DB_USER: test
       DB_PASSWORD: zaphod
       LOG_KEY: /kt/trillian-log.pem
-      SIGN_KEY: /kt/p256-key.pem
-      SIGN_KEY_PW: towel
```

Comment:
Can you also update the kubernetes [config](https://github.com/google/keytransparency/blob/master/deploy/kubernetes/keytransparency-deployment.yml.tmpl#L123-L126)? Otherwise this will break the deployed server.